Cargo.toml

```diff
@@ -28,7 +27,11 @@ unicode-normalization = "0.1.8"
 serde_json = { version = "1.0.39", optional = true }
 
 [target.'cfg(not(target_arch = "wasm32"))'.dependencies]
-rayon = "1.0.3"
+rayon = "1.1.0"
+zstd = "0.4.24+zstd.1.4.0"
```

Comment:
zstd supposedly supports wasm: https://github.com/gyscos/zstd-rs/issues/48
It needs the `wasm` feature enabled. Haven't tested this myself, so it might not work out.

Not essential to have that for this PR, either -- just noting I might tinker with this later. :D

Reply:
Ok, so I've changed it to use the wasm feature when compiling for wasm32, which at least compiles successfully. I tried adding a wasm-bindgen-test to be able to check if it _actually_ works, but running into a `Error: Cannot find module 'env'` that I'm unsure how to fix.